src/cloud-providers/src/provider.ts

```diff
@@ -253,8 +255,7 @@ export default class Provider {
    * }
    */
   async downloadMap(loadParams): Promise<{map: SavedMap; format: string}> {
-    // @ts-expect-error
-    return;
+    return Promise.reject('You must implement downloadMap');
```

Comment:
Nit: In my opinion it is cleaner to just throw an error and the async function will convert that to a rejected promise.

Reply:
yeah i can do that